src/opt.cpp

```diff
@@ -368,7 +368,7 @@ void ispc::Optimize(llvm::Module *module, int optLevel) {
 
     optPM.initFunctionPassManager();
     optPM.initLoopPassManager();
-    optPM.addLoopPass(llvm::IndVarSimplifyPass());
```

Comment:
Why is it needed? Why automatic number assignment doesn't work here?

Same question is similar change below.

Reply:
The automatic assignment works, but in order to maintain stability for lit tests which are checking for specific passes, the pass number should remain the same if edits are made earlier in the pipeline.

For this PR, this first one does not need a number, but 296 does. However, it doesn't hurt to have the extra pass numbered as well.

Reply:
The current tests no longer use the intermediate LLVMIR, so this numbering change will be removed.